app/views/home/index.html.erb

```diff
@@ -1,4 +1,26 @@
 <h3>Puntstartr</h3>
+<div class="row">
+	<div class="col s5">
+  	<h5>Trending</h5>
+  </div>
+  <div class="col s5">
+    <h5>Nearly Funded</h5>
+  </div>  
+  <div class="col s2">
+    <div class="categories">
+      <div class="card-panel teal">
+        <span class="white-text">
+	     		<h5>Categories</h5>
+	       		<% @categories.each do |category| %>
```

Comment:
another quick comment. On line 2, if you add a second class, called "container", it'll improve some of the formatting on the page.

Here's it pre-containerizing the div class:

![no container](https://cl.ly/3u3B2r0l0T3i/Puntstartr_and_1__rails_s__fsevent_watch_.jpg)

also, line 21, that `<div>` needs to be a closing div tag.

Once you make those two changes, we get: 

![after containers](https://cl.ly/15432P2A3j3a/Puntstartr.jpg)